nncf/experimental/post_training/api/engine.py

```diff
@@ -12,11 +12,12 @@
 """
 
 from abc import ABC, abstractmethod
+from typing import Callable, Dict, TypeVar
 
-from typing import Dict
-from typing import TypeVar
-from typing import Callable
+from nncf.common.tensor_statistics.collectors import \
+    TensorStatisticCollectorBase
 from nncf.experimental.post_training.api.sampler import Sampler
+from tqdm import tqdm
```

Comment:
Please move this import at the beginning according to styleguide

Reply:
Resolved.